main.go

```diff
@@ -19,6 +19,10 @@ var (
 	urgency    = kingpin.Flag("urgency", "Urgency").Default("high").String()
 	replace    = kingpin.Flag("replace", "Replace titles with regex").Strings()
 	tagFilters = kingpin.Flag("tags", "Filter PagerDuty incidents by Datadog tags").Strings()
+	// Params for uploading the report
+	confSubdomain = kingpin.Flag("subdomain", "Confluence subdomain").String()
+	spaceKey      = kingpin.Flag("space", "Confluence space key").String()
+	parentId      = kingpin.Flag("parent", "Confluence parent page id").String()
```

Comment:
It may be worth thinking about how to make it clearer, maybe call these explicitly: `confluence-subdomain`, `confluence-space`, `confluence-parent`?

Reply:
Done